freqtrade/rpc/telegram.py

```diff
@@ -331,6 +407,29 @@ def _performance(bot: Bot, update: Update) -> None:
 
 
 @authorized_only
+def _count(bot: Bot, update: Update) -> None:
+    """
+    Handler for /count.
+    Returns the number of trades running
+    :param bot: telegram bot
+    :param update: message update
+    :return: None
+    """
+    if get_state() != State.RUNNING:
+        send_msg('`trader is not running`', bot=bot)
+        return
+
+    trades = Trade.query.filter(Trade.is_open.is_(True)).all()
+
+    current_trades_count = len(trades)
```

Comment:
There is no need do declare these variables, just pass the values to format(). You can also use placeholders to make it more explicit what is being passed.